open-machine-learning-jupyter-book/ml-advanced/ensemble-learning/getting-started-with-ensemble-learning.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
<p>Same，perhaps “<strong><em>Intuition for classification ensembles</em></strong> ” would be better represented by secondary headings, rather than bolded?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ocademy-ai/machine-learning/pull/757/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
Cant use secondary headings in thr introduction part , I've tried

Reply:
What I mean by secondary headings here is that headings preceded by "##" content are automatically recognized as secondary headings, like ## Acknowledgments